data_base/SQLAlchemy.py

```diff
@@ -49,3 +41,24 @@ def is_data_model_exists(self, data_model: DB.db.Model, where_section: dict):
             exists_query = exists_query.where(data_model.__dict__[field] == value)
 
         return self.db.session.query(exists_query).scalar()
+
+    def get_entities(self, data_model, select_section: list, attributes_filters: dict, count):
```

Comment:
Add docs please,
You can add default values to you methods, default value for count here will be great